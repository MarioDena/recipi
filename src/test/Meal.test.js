import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import { BrowserRouter } from 'react-router-dom';
import Meal from '../components/meal';
import rootReducer from '../reducers/index';

const store = createStore(rootReducer);

afterEach(cleanup);

test('Contains the Meal page entirely', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Provider store={store}>
        <Meal />
      </Provider>
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
