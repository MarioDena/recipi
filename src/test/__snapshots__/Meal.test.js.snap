// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contains the Meal page entirely 1`] = `
<DocumentFragment>
  <div>
    <div
      class="container"
    >
      Loading...
    </div>
  </div>
</DocumentFragment>
`;
